bin/good-first-issue.js

```diff
@@ -27,11 +29,11 @@ cli
     }
 
     try {
-      const issues = await goodFirstIssue(input)
+      const issues = await gfi(input, options)
 
       if (issues.length === 0) {
         console.log(chalk.yellow(`\nNo Good First Issues were found for the GitHub organization, repo, or project ${chalk.white(input)}.\n`))
-        process.exit(0)
+        process.exitCode = 0
```

Comment:
I haven't seen this pattern of setting an `exitCode` - I [read it as setting the code but intentionally _not_ ending the process](https://nodejs.org/api/process.html#process_process_exitcode). Is that intentional? Is there a time when we wouldn't want to exit early?

It looks like in this case we're saying "Hey we didn't find any issues, we should exit." Given the logic below (that depends on `issues` having items, I think that's the correct behavior.

Reply:
I was told by two people I trust on Twitter that `process.exitCode` is preferable to `process.exit()`

[ref](https://twitter.com/7tonshark/status/1141135377928990722), [ref](https://twitter.com/Fishrock123/status/1141211103000576000?s=20)

The not ending is intentional. We are basically setting the exit code that will be returned _when the process exits_ but letting code finish it's thing.

Reply:
Oooh interesting, TIL. That makes sense. I think we'll need more conditionals then - as it stands, wouldn't it cause an issue here?

https://github.com/cutenode/good-first-issue/pull/146/files#diff-4762148a5cd5533dee946e78618ea789R39

I guess not because `issues.length - 1` will just be `-1`, and the `Math` stuff will work out fine; but it feels weird to run that logic even though we know it won't be used. But not a blocker 🤷‍♂

Reply:
You're 100% correct: 

![image](https://user-images.githubusercontent.com/502396/63449694-95017e00-c40e-11e9-8f0e-0ee9ffe92b83.png)

Reply:
How about this?

```
      if (issues.length === 0) {
        process.exitCode = 0
        return console.log(chalk.yellow(`\nNo Good First Issues were found for the GitHub organization, repo, or project ${chalk.white(input)}.\n`))
      }
```

Reply:
Going to ship with that. If there's further tweaks we want to make we can totally modify post-release :)

Reply:
I think that's perfect 👌 just having the `return` is a safe approach to make sure nothing breaks down the line.